client/app/libs/i18n/I18nSetup.js

```diff
@@ -0,0 +1,26 @@
+import first from 'lodash/first';
+import { addLocaleData } from 'react-intl';
+// every locale is an array of various sub locales, that follow the type defined below
+// https://github.com/yahoo/react-intl/wiki/API#addlocaledata
+/*
+  type LocaleData = {
+      locale: string,
+      [key: string]: any,
+  }
+ */
+import en from 'react-intl/locale-data/en';
+import es from 'react-intl/locale-data/es';
+import it from 'react-intl/locale-data/it';
+import nb from 'react-intl/locale-data/nb';
+import sv from 'react-intl/locale-data/sv';
+import nl from 'react-intl/locale-data/nl';
+import br from 'react-intl/locale-data/br';
```

Comment:
A reminder that we should update our i18n how-to documentation to have this step when adding new languages. I can do this once it's merged!